src/components/NewLocationPage/LocationName/utils.ts

```diff
@@ -1,5 +1,21 @@
 import { Region, State, County, MetroArea } from 'common/regions';
 
+const regionSuffixes = new Set([
```

Comment:
this can just be an array (ditto to `multipleWordSuffixIndicator` below)

const regionSuffixes = [
  'metro',
  'city',
  'county',
  'parish',
  'area',
  'borough',
]